Fable.Remoting.ClientV2/Types.fs

```diff
@@ -27,6 +28,7 @@ type RemoteBuilderOptions = {
     CustomHeaders : (string * string) list
     BaseUrl  : string option
     Authorization : string option
+    WithCredentials : bool
```

Comment:
The response doesn't really need the property, right?

Reply:
I don't believe so, I can't find any reference to it on the [response as per MDN](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/response)